konan_sdk/sdk.py

```diff
@@ -42,3 +42,14 @@ def predict(self, deployment_uuid: str, input_data: Union[Dict, str]) -> Tuple[s
         prediction_uuid, output = PredictionEndpoint(api_url=self.api_url, user=self.auth.user, deployment_uuid=deployment_uuid).post(payload=input_data)
 
         return prediction_uuid, output
+
+    def feedback(self, deployment_uuid: str, feedback: List[Dict]):
```

Comment:
I believe this needs a doc string

Reply:
see [this](https://github.com/SynapseAnalytics/konan-sdk/pull/6/files#r777225191)

Reply:
Fixed it